### DISCUSSION TITLE
Virtual scroll and multiselect

### DISCUSSION THREAD

**Discussion by zavod008** [taiga-family/taiga-ui#6702] [Q&A] [answered]
Привет ребят такой вопрос.
Я сделал мультиселект с виртуальным скролом все окей.
Но если размер инпута маленький, то появляется прокрутка ну тоже пойдет.
Я хотел это поправить впихнув tuiDropdownLimitWidth, но если его ставить в auto то выпадашка имеет ширину в 1 пиксель.
Я впринципе понимаю что ему трудно наверн определить ширину выпадашки с этим скролом, но может есть какой нибудь хак чтобы выпадашка была на всю ширину детей как в обычном мультиселекте

**Comment by zavod008** [comment]
Я сделал пример, чтобы понятней было о чем речь.
https://stackblitz.com/edit/angular-n54r8v?file=src%2Fapp%2Fapp.component.html

**Comment by waterplea** [comment, marked as answer]
Могу предложить только подхачить `max-width: 100%` на внутренний `div class="cdk-virtual-scroll-content-wrapper"`, который создаётся виртуальным скроллом. Правда так у тебя контент подрежется, он ведь скролл показывает не от жадности, а потому, что не смог слова перенести. Можешь `word-break: break-word` повесить на `tui-data-list`.

**Reply by zavod008** [reply]
Спасибо, попробую
